src/about.mdx

```diff
@@ -114,6 +114,7 @@ In addition to English and German, we want to add more languages for our user in
 Many thanks to all translators who have contributed so far:
 
 * Chinese: [Xiaocui Qiu](https://www.dla-marbach.de/ueber-uns/mitarbeiterinnen-und-mitarbeiter/alle-mitarbeiterinnen-und-mitarbeiter/detail/14621/) (DLA Marbach)
+* Farsi: Roya Zendebudie (Freie Universität Berlin)
```

Comment:
@lehkost added credits as well in the meantime. Please resolve the conflict.

Reply:
Done! :)

Reply:
You intentionally lost the link?

Reply:
Wait, there are two entries now. Only the linked one should remain I guess.